framework/lib/components/toolbox/types.ts

```diff
@@ -1,6 +1,7 @@
 import { FlexProps } from '@chakra-ui/react'
 
 export interface ToolboxProps extends Omit<FlexProps, 'direction'> {
+  isResizable: boolean
```

Comment:
This is a breaking change. After bumping Northlight, all components that use `Toolbox` will fail saying that `isResizable` property has to be specified.

Making it non-optional also requires developers to specify `isResizable` in new places where `Toolbox` is used, which can be quite annoying, as it's not a common feature used with `Toolbox`. Let's mark it as optional:
```
isResizable?: boolean
```